README.md

```diff
@@ -8,7 +8,7 @@
 # Lunes CMS
 [![Logo](.github/logo.png) Lunes - Vocabulary for your profession.](https://www.lunes.app)
 
-This is a Django 3 based content management system for the vocabulary trainer app Lunes, a project powered by the Tür an Tür [Digital Factory](https://tuerantuer.de/digitalfabrik/).
+This is a Django 3 based content management system for the vocabulary trainer app Lunes, a project powered by the Tür an Tür [Digitalfabrik](https://tuerantuer.de/digitalfabrik/).
```

Comment:
```suggestion
This is a Django 3 based content management system for the vocabulary trainer app Lunes, a project powered by [Tür an Tür – Digitalfabrik gGmbH](https://tuerantuer.de/digitalfabrik/).
```
If you want to be 100% correct, you could use the legal name of our organization :grin: